ttnn/cpp/ttnn/operations/eltwise/unary_backward/unary_backward_pybind.hpp

```diff
@@ -748,13 +751,13 @@ void bind_unary_backward_shape(
 
 template <typename unary_backward_operation_t>
 void bind_unary_backward_optional(
-    py::module& module, const unary_backward_operation_t& operation, std::string_view description, std::string_view supported_dtype = "") {
+    py::module& module, const unary_backward_operation_t& operation, const std::string_view description, std::string_view supported_dtype = "") {
```

Comment:
const std::string_view supported_dtype = ""

Reply:
I guess the code has been changes and this is from a different commit.